README.md

```diff
@@ -0,0 +1,24 @@
+# Important considerations
+----
+## LOCAL
+- To clone repository:
+```
+git clone https://github.com/repo woocommerce-apurata-payment-gateway
+```
+> Explanation: When you want to add the plugin to wordpress by cloning the repository, you must change the name of the folder from "woocommerce-acuotaz-payment-gateway" to "woocommerce-apurata-payment-gateway"
+- Change the owner:
+```
+sudo chown -R www-data:www-data woocommerce-apurata-payment-gateway
+```
```

Comment:
To be able to edit the code directly, add your user to the group www-data

used-add -G www-data # No recuerdo el comando exacto, así que búscalo porfa

And change the permissions of the folder

chmod -R g+w woocommerce-apurata-payment-gateway